pages/support.html

```diff
@@ -16,6 +16,7 @@ <h1>Commercial Support</h1>
 	
 <ul>
 <li><a href="https://objectcomputing.com/platforms/grails">Object Computing, Inc.</a> </li>
+<li><a href="https://gtit.dk/">Gennemtænkt IT A/S</a> </li>
```

Comment:
I would probably add a `target="_blank"` attribute to open a new tab, but not needed. All looks good